README.md

```diff
@@ -168,20 +168,20 @@ Code snippets like `var foo = "bar";` can be shown inline.
 Also, `this should vertically align` ~~`with this`~~ ~~and this~~.
 
 Code can also be shown in a block element.
-````
+```
```

Comment:
Here is a picture of it (Sublime): 
![blank-line](https://cloud.githubusercontent.com/assets/1694112/24479968/9328de0a-14ea-11e7-8705-29e4b6f97fcd.png)